hist/hist/src/TF1.cxx

```diff
@@ -530,13 +530,13 @@ TF1::TF1(const char *name, const char *formula, Double_t xmin, Double_t xmax, EA
       fXmax = xmin;
    }
    // Create rep formula (no need to add to gROOT list since we will add the TF1 object)
-
+   const auto formulaLength = strlen(formula);
    // First check if we are making a convolution
-   if (TString(formula, 5) == "CONV(" && formula[strlen(formula) - 1] == ')') {
+   if (TString(formula, 5) == "CONV(" && formula[formulaLength - 1] == ')') {
```

Comment:
I really meant that the `strncmp` should be included here, because that's the initial bug. 🙂 
```suggestion
   if (strncmp(formula, "CONV(", 5) == 0 && formula[formulaLength - 1] == ')') {
```